lacework/redpanda
# Internal Review - Makes rpk redpanda admin commands available on MacOS

## Cover letter

This PR fixes the following:

`rpk redpanda admin` command is unavailable on MacOS. This problem is stated in - https://github.com/redpanda-data/redpanda/issues/3666


Fixes https://github.com/redpanda-data/redpanda/issues/3666

## Release notes

* rpk redpanda admin command is now available on MacOS to interact with Remote RedPanda clusters.

## PR Testing Steps
**[This is only for Internal Reviewer to follow and test things out, we will remove this section when submitting upstream]**

- Clone the Repo
- Switch to admin_macos branch - `git checkout admin_macos`
- Go to the rpk directory - `cd src/go/rpk/`
- Run `./build.sh`

You should see a `rpk` binary under `darwin-amd64` directory.
- Run `./darwin-amd64/rpk help` and you should be able to see `redpanda admin` command now on MacOS.
- If you have not set the REDPANDA_BROKERS env var, please set it.
- You can try running `./darwin-amd64/rpk redpanda admin brokers list` command as well to see all brokers running in the your cluster.
